db/model/reviews.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const faker = require('faker');
+
+mongoose.connect('mongodb://localhost/test');// { useMongoClient: true }
+
+mongoose.Promise = global.Promise;
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connected to db');
+});
+
+var reviewSchema = new mongoose.Schema({
+  propertyName: String,
+  username: String,
+  avatar: String,
+  review: String,
+  dayPosted: Date,
+  rating: {
+    cleanliness: Number,
+    communication: Number,
+    accuracy: Number,
+    checkIn: Number,
+    location: Number,
+    value: Number
+  },
+  response: {
+    hostname: String,
+    avatar: String,
+    response: String,
+    dayCommented: Date
+  }
+});
+
+var Review = mongoose.model('Review', reviewSchema);
+// var review = mongoose.model( name: review, reviewSchema, collection 'Review');
+
+// module.exports.Review = Review;
+
+var saveData = (data) => {
+  console.log('getting to saveData');
+  console.log(data);
+  for (var i = 0; i < data.length; i++) {
+    var review = new Review({
+      username: data[i].username
+    });
+    console.log('getting to iteration');
+    review.save();
```

Comment:
one possible optimization to think about is the seeding data process is to use **Model.create**  rather than .save, it actually implements a **.save** for each document in the array, that way you don't have to looping over reviews yourself. This can eliminate one extra function so you only need to call the .create inside your seedReviews function